### PR TITLE
Update Recent Activity CSV Export File

### PR DESCRIPTION
**Pull request by mayabose**
## Description
Changes the field "Grant Assigned By" to "Grant Shared By" on the CSV file generated from the Recent Activity section in Dashboard. The change is dependent on the state of the `SHARE_TERMINOLOGY_ENABLED` environment variable.

## Testing
Generated CSV file shows the updated terminology.

## Checklist
- [ ] Provided ticket and description
- [ ] Provided testing information
- [ ] Provided adequate test coverage for all new code
- [ ] Added PR reviewers

### CODE REVIEW

**Review by jeffsmohan — APPROVED** [review]
lgtm!

**Review by jeffsmohan — CHANGES_REQUESTED** [review]
Actually, it looks like you've got a failing test, but it should be pretty quick to address. Let me know if you need any help getting the server-side tests up and running to reproduce the test failure!

File: packages/server/__tests__/api/grants.test.js
Comment (jeffsmohan) [inline packages/server/__tests__/api/grants.test.js:741]: Thanks for updating the test! Couple quick tweaks: 

1. I think this is better expressed as two separate tests (one where the flag is one, one where it's off). Otherwise, you end up duplicating the logic you're testing inside the test itself, which isn't as helpful. 
2. Looks like linting issues cropped up here again. If you're having any trouble getting autofix on save working in your editor, I'd love to help and make sure the steps get properly documented!

**Review by jeffsmohan — CHANGES_REQUESTED** [review]
Thanks for the test updates! Quick tweaks, and this should be ready to go

File: packages/server/__tests__/api/grants.test.js
Comment (mayabose) [reply packages/server/__tests__/api/grants.test.js]: After some updates/more set-up, I think linting is working now for me. Thank you for offering to help!
I split the test into two, but I'm not sure if the code after the flag and csv assignments is a bit repetitive now?

Comment (jeffsmohan) [reply packages/server/__tests__/api/grants.test.js]: Ah, good call! I don't think the repetitiveness is bad enough to block this PR if you want to land it, but I agree it's probably cleaner as three separate tests, something like this: 

```js
context('GET /exportCSVRecentActivities', () => {
  let originalShareTerminologyEnabled;

  before(() => {
    originalShareTerminologyEnabled = process.env.SHARE_TERMINOLOGY_ENABLED;
  });

  after(() => {
    process.env.SHARE_TERMINOLOGY_ENABLED = originalShareTerminologyEnabled;
  });

  it('returns valid CSV response', async () => {
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(response.statusText).to.equal('OK');
    expect(response.headers.get('Content-Type')).to.include('text/csv');
    expect(response.headers.get('Content-Disposition')).to.include('attachment');
  });

  it('includes correct column headers with share terminology disabled', async () => {
    process.env.SHARE_TERMINOLOGY_ENABLED = 'false';
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(await response.text()).to.contain('Date,Team,Grant,Status Code,Grant Assigned By,Email');
  });

  it('includes correct column headers with share terminology enabled', async () => {
    process.env.SHARE_TERMINOLOGY_ENABLED = 'true';
    const response = await fetchApi('/grants/exportCSVRecentActivities', agencies.own, fetchOptions.staff);
    expect(await response.text()).to.contain('Date,Team,Grant,Status Code,Grant Shared By,Email');
  });
});
```

**Review by jeffsmohan — APPROVED** [review]
Thanks for the test updates! Good to go as-is if you like, but I also replied to your comment about cleaning up the test code, if you wanted to tackle that.

**Review by jeffsmohan — APPROVED** [review]
lgtm!

**Pull request merged**
